vsintegration/src/FSharp.Editor/Classification/ColorizationService.fs

```diff
@@ -35,7 +35,7 @@ type internal FSharpColorizationService
 
         member this.AddSemanticClassificationsAsync(document: Document, textSpan: TextSpan, result: List<ClassifiedSpan>, cancellationToken: CancellationToken) =
             asyncMaybe {
-                let! options = projectInfoManager.TryGetOptionsForEditingDocumentOrProject(document)
+                let! options = projectInfoManager.TryGetOptionsForDocumentOrProject(document)
```

Comment:
It's 50ms on a 350 lines script. `TryGetOptionsForEditingDocumentOrProject` returns the options that was recalculated by one of the services that already use `TryGetOptionsForDocumentOrProject` (diagnostics, find usages, navigate to). So if coloring uses the fast `TryGetOptionsForEditingDocumentOrProject`. it almost always will get previous version of options because Roslyn calls classification service first (immediately after file was changed) and diagnostics - after some delay.